src/pages/Schedule.tsx

```diff
@@ -215,123 +228,161 @@ export const Schedule = () => {
       alignItems="center"
       flexDirection="column"
     >
-      <Heading mb={10}>View Schedule</Heading>
-      <Container mb={32} maxW="container.xl">
-        {!scheduleData || scheduleLoading ? (
-          <Container
-            display="flex"
-            justifyContent="center"
-            alignItems="center"
-            marginTop="4rem"
-          >
-            <Spinner size="xl" />
-          </Container>
-        ) : (
-          <>
-            <Flex alignItems="center" justifyContent="space-between" mb={5}>
-              <Select
-                id="select"
-                w="12rem"
-                value={viewState}
-                onChange={(e) => {
-                  refreshSchedule();
-                  e.target.value === "table"
-                    ? setViewState(ViewTypes.table)
-                    : setViewState(ViewTypes.calendar);
-                }}
-              >
-                <option value="table">Table View</option>
-                <option value="calendar">Calendar View</option>
-              </Select>
-              <SearchBar
-                getTermData={getScheduleRef}
-                setScheduleData={setScheduleData}
-              />
-              <Button
-                w="200px"
-                as={Link}
-                to="/schedule"
-                backgroundColor="blue.300"
-                colorScheme="blue"
-                variant="solid"
-              >
-                Submit Changes
-              </Button>
-            </Flex>
-            <Flex
-              p={10}
-              paddingTop={"0.5rem"}
-              borderRadius={10}
-              flexDir="column"
-              style={{ boxShadow: "0px 0px 30px rgba(0, 0, 0, 0.40)" }}
+      <>
+        <Heading>View Schedule</Heading>
+        <FormLabel htmlFor="year">For Year:</FormLabel>
+        <Select
+          placeholder="Select Year"
```

Comment:
```suggestion
          placeholder="Select Year"
          disabled={scheduleLoading}
```
We shouldn't allow the user to change the year while the schedule is loading